src/prasad/Stringandarray/Secondhightestnum.java

```diff
@@ -0,0 +1,42 @@
+package prasad.Stringandarray;
+/*Find second maximum from array without sorting the array.*/
+
+public class Secondhightestnum {
+	
+	int highestNum(int[]array) {
+ 		int maxNum=0;
+		for(int index=0;index<array.length;index++) {
+			int num=array[index];
+			for(int innerindex=0;innerindex<array.length;innerindex++) {
+				if(num>array[innerindex]&& num>maxNum) {
```

Comment:
Hi Prasad, this can be handle in single loop only, 
take a look of this code,
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/924/files
take of this next time